cmd/util.go

```diff
@@ -36,3 +36,18 @@ func ifFilesPassed(files []string) error {
 	}
 	return nil
 }
+
+// RemoveDuplicateFiles function will see input files and will remove duplicate entries
+func RemoveDuplicateFiles(inputFiles []string) []string {
```

Comment:
Since we are taking in `[]string` which is an underlying pointer, we can make changes to the same data structure and not return anything.

Reply:
We are not returning same `[]string` but the new one